ui/src/app/views/workspace-edit/component.spec.ts

```diff
@@ -1,58 +1,59 @@
-import {fakeAsync, TestBed} from '@angular/core/testing';
```

Comment:
Keeping this file per Neha's request, so we know what tests we need to write for the React version.